src/stories/pages/quiz.stories.tsx

```diff
@@ -1,8 +1,8 @@
-import Quiz from '../../pages/quiz';
+import TestPage from '../../pages/testPage';
 
 export default {
   title: 'Pages/Quiz',
-  component: Quiz,
+  component: TestPage,
 };
 
-export const HomePage = () => <Quiz />;
+export const HomePage = () => <TestPage />;
```

Comment:
why name is different? it is little confusing.  I you want special name, you can use alias
import TestPage as TestPageComponent  from '../../pages/testPage';

export const TestPage = () => <TestPageComponent />;